src/backends/torch/torchinputconns.cc

```diff
@@ -544,6 +639,13 @@ namespace dd
     return output;
   }
 
+  void TxtTorchInputFileConn::fill_dataset(
```

Comment:
I find this function redundant, `fill_dataset(testsets[test_id], entries)` is actually the same length as `fill_dataset(testsets, entries, test_id)` and is clearer imho

Reply:
ok, removed